feature/policy_forwarding/vrf_selection/otg_tests/multiple_vrfs_and_gue_decap/multiple_vrfs_and_gue_decap_test.go

```diff
@@ -0,0 +1,769 @@
+			flownames: []string{"1to6v4_encapped", "1to6v6_encapped", "2to7v4", "3to8v4", "4to9v4", "5to10v4", "2to7v6", "3to8v6", "4to9v6", "5to10v6"},
+		},
+		{
+			name:      "PF-2.3.3: BE1 and AF1 traffic from ATE:Port1 to ATE:Port2 simulated to be GUE Encaped and sent to the DUT's Default VRF by ATE:Port2",
+			flownames: []string{"1to6v4_encapped", "2to7v4_encapped", "1to6v6_encapped", "2to7v6_encapped", "3to8v4", "4to9v4", "5to10v4", "3to8v6", "4to9v6", "5to10v6"},
+		},
+		{
+			name:      "PF-2.3.4: BE1, AF1 and AF2 traffic from ATE:Port1 to ATE:Port2 simulated to be GUE Encaped and sent to the DUT's Default VRF by ATE:Port2",
+			flownames: []string{"1to6v4_encapped", "2to7v4_encapped", "3to8v4_encapped", "1to6v6_encapped", "2to7v6_encapped", "3to8v6_encapped", "4to9v4", "5to10v4", "4to9v6", "5to10v6"},
+		},
+		{
+			name:      "PF-2.3.5: BE1, AF1, AF2 and AF3 traffic from ATE:Port1 to ATE:Port2 simulated to be GUE Encaped and sent to the DUT's Default VRF by ATE:Port2",
+			flownames: []string{"1to6v4_encapped", "2to7v4_encapped", "3to8v4_encapped", "1to6v6_encapped", "4to9v4_encapped", "2to7v6_encapped", "3to8v6_encapped", "4to9v6_encapped", "5to10v4", "5to10v6"},
+		},
+		{
+			name:      "PF-2.3.6: BE1, AF1, AF2, AF3 and AF4 traffic from ATE:Port1 to ATE:Port2 simulated to be GUE Encaped and sent to the DUT's Default VRF by ATE:Port2",
+			flownames: []string{"1to6v4_encapped", "2to7v4_encapped", "3to8v4_encapped", "4to9v4_encapped", "5to10v4_encapped", "1to6v6_encapped", "2to7v6_encapped", "3to8v6_encapped", "4to9v6_encapped", "5to10v6_encapped"},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			validateTrafficLoss(t, ate, tc.flownames)
+		})
+	}
+}
+
+// configureDutWithGueDecap configure DUT with GUE decapsulation
+func configureDutWithGueDecap(t *testing.T, dut *ondatra.DUTDevice, guePort int, ipType string) {
+	t.Logf("Configure DUT with decapsulation UDP port %v", guePort)
+	ocPFParams := getDefaultOcPolicyForwardingParams(t, dut, guePort, ipType)
+	_, _, pf := cfgplugins.SetupPolicyForwardingInfraOC(ocPFParams.NetworkInstanceName)
+	cfgplugins.DecapGroupConfigGue(t, dut, pf, ocPFParams)
+}
+
+// getDefaultOcPolicyForwardingParams provides default parameters for the generator,
+// matching the values in the provided JSON example.
+func getDefaultOcPolicyForwardingParams(t *testing.T, dut *ondatra.DUTDevice, guePort int, ipType string) cfgplugins.OcPolicyForwardingParams {
+	return cfgplugins.OcPolicyForwardingParams{
+		NetworkInstanceName: "DEFAULT",
+		InterfaceID:         dut.Port(t, "port1").Name(),
+		AppliedPolicyName:   policyName,
+		TunnelIP:            dutlo0Attrs.IPv4,
+		GuePort:             uint32(guePort),
+		IpType:              ipType,
+		Dynamic:             true,
+		DecapProtocol:       "ip",
+	}
+}
+
+// ConfigureDUTIntf configure DUT interface IP address
+func ConfigureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
```

Comment:
this function should  not be exported for other packages to use, so use lowercase:
```suggestion
func configureDUTIntf(t *testing.T, dut *ondatra.DUTDevice) {
```

However, prefer if you use cfgplugins.NewAggregateInterface